### PR TITLE
fix: [SRM-14989]: changed the text for MS listing page according to the mocks

### PR DESCRIPTION
**Pull request by karansaraswat19**
#### Screenshots
Screenshot 2023-06-30 at 6 19 09 PM
Screenshot 2023-06-30 at 6 19 18 PM
Screenshot 2023-06-30 at 6 19 27 PM
Screenshot 2023-06-30 at 6 19 34 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by karansaraswat19** [comment]
retrigger cypress-rest

**Review by deepesh-ui — DISMISSED** [review]

**Review by praneshg239 — DISMISSED** [review]

**Comment by karansaraswat19** [comment]
retrigger jest

**Comment by karansaraswat19** [comment]
retrigger title-check

**Comment by karansaraswat19** [comment]
retrigger cypress-cv

**Comment by karansaraswat19** [comment]
retrigger cypress-cv

**Comment by karansaraswat19** [comment]
retrigger cypress-pipeline

File: src/modules/85-cv/pages/monitored-service/CVMonitoredService/__test__/CVMonitoredService.test.tsx [hunk 1]
```diff
@@ -252,6 +253,57 @@ describe('Monitored Service list', () => {
...
+        <CVMonitoredService />
+      </TestWrapper>
+    )
+
+    expect(getByText('cv.monitoredServices.youDontHaveAnyMonitoredServicesYet')).toBeInTheDocument()
+    expect(getByText('connectors.cdng.monitoredService.monitoredServiceDef')).toBeInTheDocument()
+    expect(getByText('common.useTemplate')).toBeInTheDocument()
+
+    // click add MS button
+    await userEvent.click(getByText('common.newMonitoredService'))
```
Comment (praneshg239) [inline src/modules/85-cv/pages/monitored-service/CVMonitoredService/__test__/CVMonitoredService.test.tsx:274]: why are we not asserting after click event?

File: src/modules/85-cv/components/MonitoredServiceListWidget/components/CommonMonitoredServiceListView/CommonMonitoredServiceListView.tsx [hunk 2]
```diff
@@ -230,14 +232,23 @@ const CommonMonitoredServiceListView: React.FC<CommonMonitoredServiceListViewPro
...
+          message={
+            appliedSearchAndFilter
+              ? getString('cv.monitoredServices.youHaveNoMonitoredServices')
+              : config?.module
+              ? getString('connectors.cdng.monitoredService.monitoredServiceDef')
+              : getString('cv.commonMonitoredServices.definition')
+          }
           imageClassName={css.noServiceAvailableImage}
-          containerClassName={css.noDataContainer}
+          button={!appliedSearchAndFilter ? createButton : undefined}
```
Comment (harshilbits) [inline src/modules/85-cv/components/MonitoredServiceListWidget/components/CommonMonitoredServiceListView/CommonMonitoredServiceListView.tsx:249]: Please avoid assigning undefined. Props can be passed conditionally.

File: src/modules/85-cv/components/MonitoredServiceListWidget/components/CommonMonitoredServiceListView/CommonMonitoredServiceListView.tsx [hunk 3]
```diff
@@ -230,14 +232,23 @@ const CommonMonitoredServiceListView: React.FC<CommonMonitoredServiceListViewPro
...
-      ) : content && !content.length ? (
+      ) : (
         <NoDataCard
           image={noServiceAvailableImage}
-          message={getString('cv.monitoredServices.youHaveNoMonitoredServices')}
+          messageTitle={
+            !appliedSearchAndFilter ? getString('cv.monitoredServices.youDontHaveAnyMonitoredServicesYet') : undefined
+          }
+          message={
+            appliedSearchAndFilter
```
Comment (harshilbits) [inline src/modules/85-cv/components/MonitoredServiceListWidget/components/CommonMonitoredServiceListView/CommonMonitoredServiceListView.tsx:242]: Lot of nesting here , please move it to a separate util .

File: src/modules/85-cv/pages/monitored-service/CVMonitoredService/CVMonitoredService.tsx [hunk 4]
```diff
@@ -193,54 +199,57 @@ const MonitoredService = (props: MonitoredServiceProps) => {
...
-                setEnvironment(item)
-              }}
-              className={css.filterSelect}
-            />
-            {!config ? (
-              <GridListToggle
-                initialSelectedView={selectedView}
-                onViewToggle={setSelectedView}
-                icons={{ left: 'graph' }}
+      {serviceCountData?.allServicesCount || (!serviceCountData?.allServicesCount && appliedSearchAndFilter) ? (
```
Comment (harshilbits) [inline src/modules/85-cv/pages/monitored-service/CVMonitoredService/CVMonitoredService.tsx:202]: Please move these conditions to a separate method.

File: src/modules/85-cv/components/MonitoredServiceListWidget/components/CommonMonitoredServiceListView/CommonMonitoredServiceListView.tsx (hunk 2 above)
Comment (karansaraswat19) [reply src/modules/85-cv/components/MonitoredServiceListWidget/components/CommonMonitoredServiceListView/CommonMonitoredServiceListView.tsx:249]: @harshilbits can't think of any other way. Please lmk how to do it

**Comment by karansaraswat19** [comment]
retirgger cypress-pipeline

**Comment by karansaraswat19** [comment]
retirgger cypress-pipeline

**Review by deepesh-ui — APPROVED** [review]

**Review by praneshg239 — APPROVED** [review]

**Comment by karansaraswat19** [comment]
I have covered the changes I have done, and coverage is failing for old files

**Pull request merged**